packages/babel-compat-data/scripts/build-data.js

```diff
@@ -305,11 +305,10 @@ const generateData = (environments, features) => {
         "The newly generated plugin/built-in data does not match the current " +
           "files. Re-run `npm run build-data`."
       );
-      process.exit(1);
+      process.exitCode = 1;
+      break;
     }
-
-    process.exit(0);
```

Comment:
This is rather a bug because when `--check` is provided, it will still exit after checking only `"plugin"` data. Removed after refactor.